index.bs

```diff
@@ -6571,7 +6571,9 @@ and the corresponding value is the <dfn>authenticator extension output</dfn> for
 
 <dfn>Unsigned extension outputs</dfn> are represented independently from [=authenticator data=] and returned by authenticators
 as a separate map, keyed with the same [=extension identifier=]. This map only contains entries for authenticator
-extensions that make use of unsigned outputs.
+extensions that make use of unsigned outputs. Unsigned outputs are useful when extensions output a signature over
+the [=authenticator data=] (because otherwise a signature would have to sign over itself, which isn't possible) or when
+the extension outputs should not be sent to the RP.
```

Comment:
```suggestion
the extension outputs should not be sent to the [=[RP]=].
```

Reply:
Maybe also rewrite "the outputs" to "some outputs"? (Since one extension may emit both signed and unsigned outputs)

```suggestion
some extension outputs should not be sent to the [=[RP]=].
```